dev/secrets.json.example

```diff
@@ -1,6 +1,13 @@
 {
   "adminSettings": {
-    "admins": "admin@localhost"
+    "admins": "admin@localhost,owner@localhost,cs@localhost,billing@localhost,sales@localhost",
```

Comment:
Do we want to limit these to admin and owner only?

Reply:
I wasn't involved in the design here (it was designed and already under implementation when the team restructuring took place), but my thinking is that the decision to keep `admins` as a list of all users vs. building it from the list of authorized users is for support for self-hosted instances that will not be using any RBAC.  Because of that decision, the design was additive and only applied additional role-based access controls on top of the existing authentication infrastructure.

I agree that it's not ideal.  I've captured this in https://bitwarden.atlassian.net/browse/PM-2374.